### PR TITLE
Fixing issues

### PR DESCRIPTION
**Pull request by marcell-balogh**
fixed issues detected by SonarLint

### CODE REVIEW

**Comment by Gerghet** [comment]
nice fix bro 10/10 would review this again

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -11,6 +12,7 @@
 
   // rate of failing to fire torpedos [0.0, 1.0]
   private double FAILURE_RATE = 0.0; //NOSONAR
+  Random generator = new Random();
```
Comment (Gerghet) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:15]: wow

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,18 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (Gerghet) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:35]: epic throw

**Comment by marcell-balogh** [comment]
> nice fix bro 10/10 would review this again

thanks bro im gonna merge then

**Review by Gerghet — APPROVED** [review]

**Pull request merged**